### ISSUE TITLE
EOL Giara

### ISSUE THREAD

**Issue by Newbytee** [flathub/org.gabmus.giara#4]
Reddit made the API paid and as such broke Giara. It should be marked as EOL since it's more or less useless unless Giara is granted the "accessibility exception".

**Comment by GabMus** [comment]
Thanks for opening this on my behalf, yes Giara is EOL

**Comment by Newbytee** [comment]
Yeah, but there's no flathub.json file marking it as EOL.

**Comment by bbhtt** [comment]
> yes Giara is EOL

EOL-ed as per this. If you intend to work on this app again, please open an issue on https://github.com/flathub/flathub/issues to get the repo back.
